### DISCUSSION TITLE
Why "import fpdf" rather than "import fpdf2"?

### DISCUSSION THREAD

**Discussion by vicmayrink** [py-pdf/fpdf2#1093] [General]
I'm receiving some warnings from PyCharm stating something like:

> Package containing module 'fpdf' is not listed in the project requirements

Thus, I'm wondering why should we run `import fpdf` rathen than `import fpdf2`?

Suppose a large project with legacy code using the original fpdf package and new codes using fpdf2. It should be quite hard to differentiate which code is using each version since the module name is the same when importing.

**Comment by Lucas-C** [comment]
Hi @vicmayrink

There has been some discussion about this in this issue: https://github.com/py-pdf/fpdf2/issues/925
I invite you to read this thread for some context on the subject.

Now, we could consider allowing `from fpdf2 import FPDF`.
If it's doable in a single Pypi package (I'm not sure), maybe that could be a good idea.

**Reply by vicmayrink** [reply]
That's an interesting discussion, thanks for your reply :)
